src/Components/LandingPage/ServiceUnavailableAlert.tsx

```diff
@@ -1,13 +1,29 @@
 import React from 'react';
 
 import { Alert } from '@patternfly/react-core';
+import { useVariant } from '@unleash/proxy-client-react';
 
 const ServiceUnavailableAlert = () => {
+  const variant = useVariant('image-builder.service-unavailable');
+  const payload = variant.payload
+    ? JSON.parse(variant.payload.value)
+    : undefined;
```

Comment:
I assume `undefined` results in the payload not being rendered and prevents rendering issues?

Reply:
Indeed. Had to add some optional chaining for good measure, but the default alert if the payload is not defined should look like this:
<img width="997" height="164" alt="image" src="https://github.com/user-attachments/assets/c44d2ee9-1c77-4794-8b5c-999f23ce73a5" />